slither/tools/mutator/utils/file_handling.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import Dict, List
+import logging
+
+logger = logging.getLogger("Slither-Mutate")
+
+duplicated_files = {}
+
+# function to backup the source file 
```

Comment:
Here and other places, cane we use python docstrings https://peps.python.org/pep-0257/